code/components/voip-server-mumble/src/server.cpp

```diff
@@ -551,26 +575,6 @@ static InitFunction initFunction([]()
 				return;
 			}
 
-			// is this a Mumble ping?
-			if (len == 12 && *(uint32_t*)data == 0)
```

Comment:
Just to be sure. The ping was only required for external connections?

Reply:
Yes, this is only used for external clients to show the ping/user count on the "Mumble Server Connect" list

![image](https://github.com/user-attachments/assets/efc445b0-5ffe-46f9-8459-a41520f65769)

Us not providing this won't matter for proper clients, all of this information will be sent via [`ServerSync`](https://github.com/citizenfx/fivem/blob/52f3489dd3764ccbc58cb8ccf2ca21f711257872/code/components/voip-server-mumble/src/Mumble.proto#L100-L111) packet